corehq/apps/app_manager/static/app_manager/js/forms/case_config_ui.js

```diff
@@ -72,9 +72,11 @@ hqDefine('app_manager/js/forms/case_config_ui', function () {
                         self.requires(requires);
                         self.setPropertiesMap(data.propertiesMap);
 
-                        hqImport('analytix/js/kissmetrix').track.event("Saved question as a Case Property", {
-                            questionsSaved: data.propertiesMap.case.length,
-                        });
+                        if (_(data.propertiesMap).has(self.caseType)) {
```

Comment:
@orangejenny I've implemented changes that should always get the proper number of questions in a case by case type.

Reply:
Hi Marcin, the count seems to always be `1` for me, I'm wondering if there's something amiss with the `property` call. This is my debugger locally:

<img width="696" alt="screen shot 2018-02-13 at 1 34 43 pm" src="https://user-images.githubusercontent.com/1486591/36167120-bef0e594-10c2-11e8-9229-f60715db755b.png">